lib/decidim/decidim_awesome/custom_fields.rb

```diff
@@ -60,6 +64,27 @@ def apply_to_first_textarea
         textarea["userData"] = [xml]
         textarea["name"]
       end
+
+      def translate_values!
+        deep_transform_values!(@fields) do |value|
+          next value unless value.is_a? String
+          next value unless (match = value.match(/^(.*\..*)$/))
```

Comment:
```suggestion
          next value unless (match = value.match(/^(decidim\..*)$/))
```

getting back to your idea of marking the if you want to translated or no, maybe we could just force to start with `decidim.` any thing that needs to be translated? what you think?

Reply:
Hmm... that would work. How about another prefix that isn't as loaded, like `custom_fields`?

Reply:
yes, however,... do we want to distinct between translation keys or just any direct text? can't we just write the key if no translation is available? I mean, ActiveRecord works just this way. 
I we do this approach we leave to the user how "tidy" they want to be.

I am not concerned about people using a key as a final text, it seems to me that this is a very improbable case, and even in this case you can simply create the key.

Reply:
I am insisting in this way to do thinkg so we don't have to force or explain how to deal with it, it "just works"

Reply:
And, finally, we could use `decidim.` as a default and add a configuration option to let the developer decide a different thing if they want (even if they want to leave it empty). So everybody happy

Reply:
Ah, I'm ok with not using a marker at all... That's how I've handled it now and I like it.